modules/s3-bucket/main.tf

```diff
@@ -52,7 +66,7 @@ module "s3_bucket" {
 
   # Static website configuration
   cors_rule_inputs = var.cors_rule_inputs
-  website_inputs   = var.website_inputs
+  # website_inputs   = var.website_inputs
```

Comment:
Consider taking over/finishing https://github.com/cloudposse/terraform-aws-s3-bucket/pull/142 so we can have this module support websites. Otherwise, this lack of support needs to be prominently documented.